src/test/java/core/basesyntax/dao/StorageDaoImplTest.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax.dao;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+import core.basesyntax.db.Storage;
+import java.util.HashMap;
+import org.junit.Before;
+import org.junit.Test;
+
+public class StorageDaoImplTest {
+    private StorageDao storageDao;
+
+    @Before
+    public void setUp() {
+        storageDao = new StorageDaoImpl();
+        Storage.fruitsQuantity.clear();
+    }
+
+    @Test
+    public void add_fruitWithQuantity_OK() {
+        String fruitName = "banana";
+        int quantity = 100;
+        HashMap<String, Integer> result = storageDao.add(fruitName, quantity);
```

Comment:
I'm a little confused why method for adding a new element returns the whole storage